src/tracer.js

```diff
@@ -184,6 +184,8 @@ export default class Tracer {
             startTime = Utils.getTimestampMicros();
         }
 
+        // This boolean ensures that a parent with reference type 'follows_from'
+        // is selected when no childOf is passed in.
```

Comment:
the opposite - it ensures that CHILD_OF reference is preferred as a parent even if it comes after FOLLOWS_FROM